src/js/jwt/jwt.js

```diff
@@ -235,6 +237,20 @@ exports.getUserInfo = () => {
         }
     });
 };
+// Challenge auth and login if the user could be logged in, but in an unauth state
+exports.challengeAuth = () => {
+    log('Challenging Auth');
+    //return refreshTokens();
+    priv.keycloak.login({onLoad : "login-required"})
```

Comment:
You should return this promise, otherwise this is going to return `undefined`

Reply:
fixed

Reply:
fixed